content/en/blog/_posts/2024-07-22-feature-gates_client-go.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: blog
+title: 'Introducing Feature Gates to Client-Go: Enhancing Flexibility and Control'
+date: 2024-07-22
+slug: feature-gates-in-client-go
+author: >
+Ben Luddy (Red Hat),
+Lukasz Szaszkiewicz (Red Hat)
```

Comment:
Looks like the PR build preview failed [(logs here)](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/669e4be3845d380008c64d86) with following error:

>1:10:04 PM: Error: error building site: assemble: "/opt/build/repo/content/en/blog/_posts/2024-07-22-feature-gates_client-go.md:7:1": failed to unmarshal YAML: yaml: line 7: could not find expected ':'
1:10:04 PM: make: *** [Makefile:43: deploy-preview] Error 1

I have provided a correction in the suggestion below. Hope it helps with the build error.

```suggestion
author: >
  Ben Luddy (Red Hat),
  Lukasz Szaszkiewicz (Red Hat)
```